lxd/device/nic.go

```diff
@@ -14,45 +14,582 @@ import (
+		//
+		// ---
+		//  type: string
+		//  managed: no
+		//  shortdesc: MAAS IPv6 subnet to register the instance in
+		"maas.subnet.ipv6": validate.IsAny,
+		// lxdmeta:generate(entities=device-nic-bridged; group=device-conf; key=ipv4.address)
+		// Set this option to `none` to restrict all IPv4 traffic when {config:option}`device-nic-bridged-device-conf:security.ipv4_filtering` is set.
+		// ---
+		//  type: string
+		//  managed: no
+		//  shortdesc: IPv4 address to assign to the instance through DHCP
+
+		// lxdmeta:generate(entities=device-nic-ovn; group=device-conf; key=ipv4.address)
+		//
+		// ---
+		//  type: string
+		//  managed: no
+		//  shortdesc: IPv4 address to assign to the instance through DHCP
+
+		// lxdmeta:generate(entities=device-nic-ipvlan; group=device-conf; key=ipv4.address)
+		// Specify a comma-delimited list of IPv4 static addresses to add to the instance.
+		// In `l2` mode, you can specify them as CIDR values or singular addresses using a subnet of `/24`.
+		// ---
+		//  type: string
+		//  shortdesc: IPv4 static addresses to add to the instance
+
+		// lxdmeta:generate(entities=device-nic-routed; group=device-conf; key=ipv4.address)
+		// Specify a comma-delimited list of IPv4 static addresses to add to the instance.
+		// ---
+		//  type: string
+		//  shortdesc: IPv4 static addresses to add to the instance
+		"ipv4.address": validate.Optional(validate.IsNetworkAddressV4),
+		// lxdmeta:generate(entities=device-nic-bridged; group=device-conf; key=ipv6.address)
+		// Set this option to `none` to restrict all IPv6 traffic when {config:option}`device-nic-bridged-device-conf:security.ipv6_filtering` is set.
+		// ---
+		//  type: string
+		//  managed: no
+		//  shortdesc: IPv6 address to assign to the instance through DHCP
+
+		// lxdmeta:generate(entities=device-nic-ovn; group=device-conf; key=ipv6.address)
+		//
+		// ---
+		//  type: string
+		//  managed: no
+		//  shortdesc: IPv6 address to assign to the instance through DHCP
+
+		// lxdmeta:generate(entities=device-nic-ipvlan; group=device-conf; key=ipv6.address)
+		// Specify a comma-delimited list of IPv6 static addresses to add to the instance.
+		// In `l2` mode, you can specify them as CIDR values or singular addresses using a subnet of `/64`.
+		// ---
+		//  type: string
+		//  shortdesc: IPv6 static addresses to add to the instance
+
+		// lxdmeta:generate(entities=device-nic-routed; group=device-conf; key=ipv6.address)
+		// Specify a comma-delimited list of IPv6 static addresses to add to the instance.
+		// ---
+		//  type: string
+		//  shortdesc: IPv6 static addresses to add to the instance
+		"ipv6.address": validate.Optional(validate.IsNetworkAddressV6),
+		// lxdmeta:generate(entities=device-nic-bridged; group=device-conf; key=ipv4.routes)
+		// Specify a comma-delimited list of IPv4 static routes to add on the host to the NIC.
+		// ---
+		//  type: string
+		//  managed: no
+		//  shortdesc: IPv4 static routes to add on host to NIC
```

Comment:
What is this supposed to mean? Maybe "IPv4 static routes to the NIC to add on the host"?

Reply:
The way I understand it is that this setting requires a list of IPv4 static routes (specified in a comma-separated format) that should be added on the host system for this NIC device.

Reply:
If this is correct, I like the wording you suggested.

Reply:
fixed